src/pages/Quizpage/Quizpage.tsx

```diff
@@ -0,0 +1,112 @@
+import './quizpage.css';
+import { useLocation } from 'react-router'
+import { QuestionBox, Header, Timer } from '../../components/index'
+import { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom';
+import { RequestApi } from './RequestApi'
+import { quizQuestions, ServerData, ServerError } from './quiz.types'
+import { useQuiz } from '../../Context/QuizContext'
+
+export function Quizpage() {
+  const { currentQueNo, score, dispatch } = useQuiz()
+  const query = new URLSearchParams(useLocation().search).get("page")
+  const [serverData, setServerData] = useState<ServerData | null>(null)
+  const [serverFailure, setServerFailure] = useState<ServerError | null>(null)
+  const [question, setQuestion] = useState<quizQuestions | null>(null)
```

Comment:
Just a suggestion .. Can do a reducer for question, setQuestion , name - setName etc.

Reply:
I did make a reducer which in the context folder to handle it, on click on the next button the reducer function increments the question number.